docs/standards/rest.md

```diff
@@ -115,6 +115,55 @@ headers:
 
 [Header field names are case insensitive](https://datatracker.ietf.org/doc/html/rfc7230#section-3.2). Snyk v3 API specs must use kebab case for consistency. All non-standard headers that are unique to Snyk must begin with `snyk-` (e.g. `snyk-requested-version`).
 
+## <a id="request-parameters"></a>Request Parameters
+
+### <a id="formats"></a>Formats
+
+Some resource types may be expressed with a media content-type format other than JSON API. These alternative formats may be offered with these format request parameters:
+
+#### `format` query parameter
+
+This is a URL query parameter of the form `?format=format_name`, where _format_name_ is an application-specific short nickname for the media content-type. This parameter's schema must be defined as an enum.
+
+For example, if we offered a SARIF representation of a single resource, the format query parameter might be `sarif`, while the `Content-Type` is `application/sarif+json`.
+
+#### `Accept` request header
+
+This is a request header of the form `Accept: content-type`, where _content-type_ is an IANA assigned or proposed content media type.
+
+Continuing with the above example, `Accept: application/sarif+json`.
+
+#### Response
+
+The response to a requested format may be:
+
+__200 OK__: The `Content-Type` response header indicates the requested format. The response body contains content in the requested format.
+
+Public REST API endpoints must provide a JSON API response.
```

Comment:
Is this in conflict with the above and below? Perhaps:

_Public REST API endpoints must provide a JSON API response, unless another response format is explicitly requested._